.github/workflows/cd.dev.yml

```
@@ -195,7 +196,7 @@ jobs:
           SERVER_DEST_BASE="${{ env.SERVER_USER }}@${{ env.SERVER_HOST }}:${{ env.DEPLOY_DIR }}"
 
           # mkdir
-          ${{ env.SSH_CMD }} "mkdir -p ${{ env.DEPLOY_DIR }}" 
+          ${{ env.SSH_CMD }} "mkdir -p ${{ env.DEPLOY_DIR }}-dev" 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)